R/deploymentTarget.R

```diff
@@ -18,11 +18,10 @@ deploymentTarget <- function(appPath = ".",
   if (nrow(appDeployments) == 0) {
     fullAccount <- findAccount(account, server)
     if (is.null(appName)) {
-      appName <- generateAppName(appTitle, appPath, account, unique = FALSE)
+      appName <- generateAppName(appTitle, recordPath, account, unique = FALSE)
```

Comment:
This reinforces that the `recordPath` really should be original source of the content.